docs/migration/5_0.md

```diff
@@ -381,3 +381,7 @@ The following keys have been added to `configurator-common.ts`:
 #### Template changes
 
 - Top element `<div>` has been changed to `<sector>` to make accesibility navigation consistent. 
+
+### BadRequestHandler
+
+- Moved `handleVoucherOperationError` method to new handler, `BadVoucherRequestHandler`
```

Comment:
Moved `handleVoucherOperationError` method from `core` to the new handler `BadVoucherRequestHandler` in `cart-lib`.